README.md

```diff
@@ -0,0 +1,252 @@
+          model: oasis
+          ncpus: 0
+          input:
+            - /short/public/access-esm/payu/input/common/coupler
+
+This is probably the meatiest part of the configuration, so let's look at it in more detail.
+
+Each submodel has
+- a **name**
+- the **model** to know which driver to use
+- the number of CPUs that this model should receive (**ncpus**)
+- the location of the executable to use (**exe**)
+- one or more locations for the **input** files.
+
+The **name** is more than a useful reminder of what the model is.
+PAYU expects this submodel's configuration files in a subdirectory with that name.
+
+    collate:
+       exe: /short/public/access-esm/payu/bin/mppnccombine
+       restart: true
+       mem: 4GB
+
+Ask Aidan
+
+    restart: /short/public/access-esm/payu/restart/pre-industrial
+
+Ask Aidan
+
+    calendar:
+        start:
+            year: 101
+            month: 1
+            days: 1
+
+        runtime:
+            years: 1
+            months: 0
+            days: 0
+
+Here is the start date, and the runtime **per run**.
+The total time you want to model is `runtime` * `number of runs`
+
+    runspersub: 5
+
+This `runspersub` feature is a nifty tool to allow you to bundle several runs into a single submission for the PBS queue.
+
+Let's have an example: Say you told payu to make 7 runs with the above setting.
+Each run would have a runtime of 1 year. So in the first submission it would run the model 5 times, to model years 101 through 105 respectively.
+
+Then it would automatically resubmit another pbs job to model years 106 and 107, and then end.
+
+### Setting up the Atmosphere Submodel
+
+The **name** in `config.yaml` for the atmosphere submodel is "atmosphere", so the configuration of the UM will be in the `atmosphere` subdirectory.
+
+    $ ls atmosphere/
+    CNTLALL   SIZES      __pycache__  ftxx       ihist          prefix.CNTLATM
+    CONTCNTL  STASHC     cable.nml    ftxx.new   input_atm.nml  prefix.CNTLGEN
+    INITHIS   UAFILES_A  errflag      ftxx.vars  namelists      prefix.PRESM_A
+    PPCNTL    UAFLDS_A   exstat       hnlist     parexe         um_env.py
+
+There are many configuration files, but I want to note the `um_env.py`.
+This file is used to set environment variables for the UM.
+The UM driver of PAYU will look for this file and execute it.
+
+### Setting up the Ocean Submodel
+
+The **name** in `config.yaml` for the ocean submodel is "ocean", so the configuration
+of MOM will be in the `ocean` subdirectory.
+
+    $ ls ocean
+    data_table  diag_table  field_table  input.nml
+
+
+### Setting up the Ocean Submodel
+
+The **name** in `config.yaml` for the ice submodel is "ice", so the configuration
+of CICE will be in the `ice` subdirectory.
+
+    $ ls ice/
+    cice_in.nml  input_ice.nml
+
+## Running the Model
+
+If you have set up the modules system to use the `/g/data3/hh5/public/modules` folder, a simple `module load payu` should give you access to the payu system.
```

Comment:
See above

Reply:
I have mentioned this above, but I thought repeating it here in case someone came straight down to the running section. Or do you mean something else with 'see above'?

Reply:
See above as it is incorrect. We're not going to have a separate payu module but include it in conda